addons/l10n_ar_website_sale/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name': 'Argentinean Website',
+    'version': '1.0',
+    'category': 'Localization',
```

Comment:
This category needs adapting (check other modules in v14)

Reply:
@jco-odoo Accounting/Localizations/Website it's ok? I've already changed

Reply:
Because it is you.